src/constants.ts

```diff
@@ -2,27 +2,125 @@ import { Endpoints } from './models/core/Endpoints';
 
 export const defaultApiVersion = 20220511;
 
-export const defaultEndpoints: Required<Endpoints> = {
-  universalSearch: 'https://liveapi.yext.com/v2/accounts/me/search/query',
-  verticalSearch: 'https://liveapi.yext.com/v2/accounts/me/search/vertical/query',
-  questionSubmission: 'https://liveapi.yext.com/v2/accounts/me/createQuestion',
-  status: 'https://answersstatus.pagescdn.com',
-  universalAutocomplete: 'https://liveapi-cached.yext.com/v2/accounts/me/search/autocomplete',
-  verticalAutocomplete: 'https://liveapi-cached.yext.com/v2/accounts/me/search/vertical/autocomplete',
-  filterSearch: 'https://liveapi-cached.yext.com/v2/accounts/me/search/filtersearch',
-} as const;
+export enum CloudRegion {
+  US = 'us',
+  EU = 'eu',
+}
+
+export enum Environment {
+  PROD = 'prod',
+  SANDBOX = 'sbx',
+}
```

Comment:
can you add some docs around these? They should be publicly exported so there will be documentation generated for them

Reply:
Done